GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+or other means.
+
+### Contributor Responsibilities
+
+- Follow the [cert-manager Code of Conduct](https://github.com/cert-manager/cert-manager/blob/master/CODE_OF_CONDUCT.md).
+- Follow the guidelines in [the Contributing page](https://cert-manager.io/docs/contributing/).
+
+## GitHub Member
+
+GitHub Members are active contributors to the cert-manager project, or one of
+the related projects in the cert-manager GitHub organisation.
+
+A contributor is considered to be active when they have had at least one
+interaction (comment on an issue or PR or message in the `#cert-manager` or
+`#cert-manager-dev` channels in the Kubernetes Slack) within the last 18 months.
+
+Members that have been inactive over the past 18 months may be removed from the
+GitHub organization.
+
+**Defined by:** Member of the cert-manager GitHub organization.
+
+### Becoming a GitHub Member
+
+To be added as a GitHub member of the cert-manager organization, you will need
+to look for two sponsors with at least the `reviewer` role. These two sponsors
+must have had some meaningful interaction with you on an issue on GitHub or on
+the `#cert-manager` or `#cert-manager-dev` channels on the Kubernetes Slack.
+
+Then, open an issue on the [community][] repository and mention your sponsors as
+well as links to the meaningful interactions (Slack threads, GitHub issues). Ask
+your sponsors to confirm their sponsorship by commenting on your PR. After that,
+your request will be reviewed by a cert-manager admin, in accordance with their
+SLO.
+
+To be added as a GitHub member, you will also need to enable [two-factor
+authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) on your GitHub account.
+
+GitHub members are encouraged to engage with the mailing list [cert-manager-dev@googlegroups.com](https://groups.google.com/g/cert-manager-dev) as well as the [`#cert-manager`](https://kubernetes.slack.com/messages/cert-manager) and [`#cert-manager-dev`](https://kubernetes.slack.com/messages/cert-manager-dev) channels on the Kubernetes Slack.
+
+### GitHub Member Responsibilities
+
+No extra responsabilities.
+
+## Reviewer
+
+The mission of the reviewer is to read through PRs for quality and correctness
+on all or some part of cert-manager. Reviewers are knowledgeable about the
+codebase as well as software engineering principles. Individuals with expertise
+in documentation, website content, and other facets of the project are also
+encouraged to join as reviewers.
+
+**Defined by:** the `reviewers` section in the file [`OWNERS`](./OWNERS).
+
+### Becoming a Reviewer
+
+To become a reviewer, you will need to look for a sponsor with at least the
+approver role. Your sponsor must have had close interactions with you: they must
+have closely reviewed one of your PRs or worked with you on a complex issue.
+
+Then, create a PR on the [community][] repository to add your name to the list
```

Comment:
@maelvls the PR should be in the repo itself, not on the community repo.

Reply:
Thanks for catching this!